src/vue/modules/document-upload-form/store/index.js

```diff
@@ -0,0 +1,91 @@
+import { types } from './types'
```

Comment:
you need store basically in 2 cases:
 * when you have multiple actors that can modify the data
 * when you have multiple readers of that data

here you have only `document-upload-form` that writes/reads the data stored here

Reply:
Moved store logic to the mixin